samples/__snapshots__/reads.js

```diff
@@ -0,0 +1,147 @@
+exports['reads should read one row 1'] = `
+Reading data for phone#4c410523#20190501:
+Column Family stats_summary
+\tconnected_cell: 1 @1559361600000000
+\tconnected_wifi: 1 @1559361600000000
+\tos_build: PQ2A.190405.003 @1559361600000000
+
+
+`;
+
+exports['reads should read part of one row 1'] = `
```

Comment:
I really appreciate how this abstracts the fixtures from the data, thank you :+1: